src/autorest-core/package.json

```diff
@@ -1,10 +1,7 @@
 {
-  "name": "@microsoft.azure/autorest-core",
```

Comment:
this file was reverted to an earlier state, important information is lost! make sure you have a clean commit history, that should resolve issues like these - manually checking all these changes here for regressions will be a nightmare.

Actually, the PR you closer earlier was clean! I last rebased it on top of `2.0` *today*. What caused you to reopen it like this? AFAIK all that was missing was moving the docs folders and adjusting the core to acquire the plugin dynamically instead of statically.